src/lib/assets/logos/FrequencyLogo.svelte

```diff
@@ -8,6 +8,5 @@
   <path
     d="M216.3 28.5v-8.6h-10.5v49.9H217V44c0-9.7 4.2-14.2 13.2-14V19c-7.1.3-11.8 3.5-14 9.3zM118.8 0H7a7 7 0 0 0-7 7v97.8a7 7 0 0 0 3.3 6 7 7 0 0 0 6.8.3L94 69a7 7 0 0 0-3.2-13H36.6L122 13.3A7 7 0 0 0 118.8 0Zm35 69.8h11.7V40.3h35v-9.8h-35V9.9h35V0h-46.7v69.8Zm103.6-50.6a25 25 0 0 0-25.8 25.7c0 15 10.7 25.7 26 25.7s21-7 24-17.1h-10.3a13.7 13.7 0 0 1-13.2 8.2c-9.7 0-16-6.6-16.2-16.2h40.3c0-14.5-8.6-26.1-25-26.1zm-14.3 19a15.1 15.1 0 0 1 14.6-10.5c6.9 0 12 4 13.9 10.5h-28.4Zm232.5-19a16.1 16.1 0 0 0-14.6 8.1V20h-10.4v49.9h11.2v-29c0-7.1 3.8-12.1 10.4-12.1 6.7 0 8.7 4.4 8.7 11.7V70h11.3V38.3c0-11.6-6.5-19-16.6-19zm62.8 30.3c-1.3 7-6.4 11.2-14.1 11.2a16 16 0 0 1-15.6-16c0-9.4 6.5-15.8 15.6-15.8s12.8 4.4 14 11.1h11c-1.6-12.5-11-20.8-25-20.8a26 26 0 0 0-26.5 25.7c0 15.1 11 25.7 26.5 25.7s23.5-8.2 25-21h-11zM588.2 20l-14.9 24.8L559.5 20h-12l19.7 35-19.8 33.2h11.8L600 20Zm-168.5-.7A25 25 0 0 0 393.8 45c0 15 10.7 25.7 26 25.7 15.4 0 21-7 24-17.1h-10.3a13.7 13.7 0 0 1-13.1 8.2c-9.7 0-16-6.6-16.2-16.2h40.3c0-14.5-8.6-26.1-25-26.1zm-14.3 19A15.1 15.1 0 0 1 420 27.7c7 0 12 4 13.9 10.5h-28.4Zm-28.3 12c0 6.6-3.6 10.5-9.7 10.5-6 0-9.7-3.9-9.7-10.5V19.9h-11.2v30.9c0 12 8.2 19.7 21 19.7 12.6 0 20.8-7.8 20.8-19.7v-31H377Zm-49.4-20.5a20.4 20.4 0 0 0-18.5-10.5c-13.5 0-23.3 10.5-23.3 25.7 0 15.1 9.7 25.7 23.3 25.7 13.6 0 14-3.5 17.7-9.3v27.1h11.3V20.1h-10.5v9.7zm-16 31.2c-9 0-14.9-6.8-14.9-16.2 0-9.4 6.2-16 14.9-16s15.2 6.5 15.2 16S320.6 61 311.7 61z"
     fill="currentColor"
-    class="transition-all duration-1000"
```

Comment:
worried that this was there for a reason.

Reply:
As far as I can tell, this was a copy paste issue from my work cleaning up the svg. It appeared on the other ones. So somewhere I have a tool adding it. We do not need it.